src/components/v2/V2Create/forms/TokenForm/ReservedTokensFormItem.tsx

```diff
@@ -46,8 +46,7 @@ export default function ReservedTokensFormItem({
         checked={reservedRateChecked}
         onToggled={checked => {
           setReservedRateChecked(checked)
-          setReservedRate(0)
-          onChange(0)
+          if (!checked) onChange(0)
```

Comment:
probably should source this from `defaultFundingCycleMetadata.reservedRate`, wdyt?